core/src/main/java/org/infinispan/container/offheap/MemoryAddressHash.java

```diff
@@ -70,9 +59,27 @@ public LongStream toStream() {
     * @return stream with the valid memory pointers to stored values
     */
    public LongStream toStreamRemoved() {
-      return LongStream.iterate(memory, l -> l + 8)
+      return LongStream.iterate(0, l -> l + 8)
```

Comment:
Not sure what the `removed` in the name stands for, but regular `toStream()` isn't used, so you could rename `toStreamRemoved` to `toStream`.

Reply:
The removed is that it removes the address from the lookup table as it iterates over it (note it invokes `getAndSetLong` passing a 0. We definitely don't want to call it just `toStream`.

Reply:
I didn't notice, then I would call it `removeAll()` instead.

Reply:
Sure.